keras_app/views/layers_export.py

```diff
@@ -414,6 +414,7 @@ def recurrent(layer, layer_in, layerId):
     use_bias = layer['params']['use_bias']
     dropout = layer['params']['dropout']
     recurrent_dropout = layer['params']['recurrent_dropout']
+    return_sequences = True
```

Comment:
If you took in this value during import, why did you hardcode it for export? I am changing it for you.

Reply:
@utsavgarg Sorry I forgot to commit that changed file :sweat_smile: